src/lib/gui/tls/TlsCertificate.cpp

```diff
@@ -37,6 +37,8 @@ bool TlsCertificate::generateCertificate(const QString &path, int keyLength) con
     return false;
   }
 
+  qInfo() << "TLS PATH " << path.toStdString();
```

Comment:
Please use correct case and colon to delimit values. Space at the end isn't needed, Qt adds that automatically with `<<`

```suggestion
  qInfo() << "tls path:" << path.toStdString();
```

Adding a space introduces a double-space:
```
[2025-08-26T10:58:53] INFO: TLS PATH  "/home/nick/.config/Deskflow/tls/deskflow.pem"
```

Reply:
Chances are if you see an ALL CAPS message it should have been removed as it a testing message for myself.